Makefile

```diff
@@ -99,6 +99,14 @@ test: .coverage
 			-v \
 			./pfcpiface
 
+p4-constants:
+	$(info *** Generating go constants...)
+	@docker run --rm -v $(CURDIR):/app -w /app \
+		opennetworking/up4-ptf:latest python ./utils/go_gen_p4_const.py \
+		--output internal/p4constants/p4constants.go --p4info conf/p4/bin/p4info.txt
+	@docker run --rm -v $(CURDIR):/app -w /app \
+		golang:1.16-alpine gofmt -w internal/p4constants/p4constants.go
```

Comment:
hardcoding everything since I do not see any value in using variables here. Let me know if you think otherwise